Fatras/include/ActsFatras/Digitization/UncorrelatedHitSmearer.hpp

```diff
@@ -0,0 +1,141 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <exception>
+#include <system_error>
+
+#include "Acts/EventData/Measurement.hpp"
+#include "Acts/EventData/ParameterSet.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Surfaces/SurfaceError.hpp"
+#include "Acts/Utilities/Helpers.hpp"
+#include "Acts/Utilities/ParameterDefinitions.hpp"
+#include "Acts/Utilities/Result.hpp"
+#include "ActsFatras/EventData/DigitizedHit.hpp"
+#include "ActsFatras/EventData/Hit.hpp"
+
+namespace ActsFatras {
+
+/// Smearing functions definition, it retuns the smeared parameter and the
+/// (approximated) error for the covariance entry
+using SmearFunction = std::function<Acts::Result<std::pair<double, double>>()>;
+
+/// Parameter smearer for fast digitisation that can produce surface bound
+/// and free measurements.
+///
+/// This smearer only supports uncorrelated smearing of parameters
+class UncorrelatedHitSmearer {
+ public:
+  /// @brief smearing content
+  class Content : public DigitizedHit::IContent {};
+
+  /// Shorthand for Measurement on Surfaces (bound parameterisation)
+  template <Acts::BoundIndices... params>
+  using SurfaceMeasurement =
+      Acts::Measurement<DigitizedHit, Acts::BoundIndices, params...>;
+
+  /// Generic implementation of a smearing meathod
+  /// for surface based Measurements. This can be applied with custom
+  /// smearing functions and to variant type measurements.
+  ///
+  /// @tparam hit_t definition requires position(), unitDirection(), time()
```

Comment:
We don't have the `hit_t`, so this comment could we removed?

Reply:
Left-over, removing it.